state-machine/src/main/java/team/techtigers/actions/ParallelAction.java

```diff
@@ -1,42 +1,43 @@
 package team.techtigers.actions;
 
 /**
- * Allows multiple servo actions to be performed at once
+ * Allows multiple actions to be performed at once
  */
 public class ParallelAction implements IAction {
     private final IAction[] actions;
 
     /**
      * Initializes all of the values
+     *
      * @param actions array of actions
      */
     public ParallelAction(IAction... actions) {
-        if (actions == null){
+        if (actions == null) {
             throw new IllegalArgumentException("Invalid actions (arg #1)");
         }
 
         for (int i = 0; i < actions.length; i++) {
             IAction action = actions[i];
-            if (action == null){
+            if (action == null) {
                 throw new IllegalArgumentException("Invalid. Index: " + i);
```

Comment:
Make the error more descriptive (e.g. Null Action at index ___), users of the library may not know what invalid is  referring to/why it is happening